dev/docker/Dockerfile.rpi

```diff
@@ -0,0 +1,73 @@
+FROM armhf/debian:jessie
+
+# clean and update dependencies
+RUN apt-get clean && apt-get update
+
+# get locale dependency
+RUN apt-get install locales
+
+# set locales
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8  
+ENV LANGUAGE en_US:en  
+ENV LC_ALL en_US.UTF-8 
+
+RUN rm -vfr /var/lib/apt/lists/*
+RUN apt-get -y update
+
+# system dependencies
+RUN apt-get -y install\
+        git\
+        nginx\
+        python-software-properties\
+        software-properties-common\
+        supervisor\
+        curl
+
+# python requirement dependencies
+RUN apt-get -y install\
+        # dependencies for ansible's dependency module "cryptography"
+        build-essential\
+        libssl-dev\
+        libffi-dev
+
+# download python 3.6 source
+RUN curl https://www.python.org/ftp/python/3.6.0/Python-3.6.0.tgz -o Python-3.6.0.tgz -s && tar -xzvf Python-3.6.0.tgz
+RUN cd Python-3.6.0/ && ./configure && make -j 2 && make install
+
+RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py | python3.6 -
```

Comment:
```shell
curl https://example.com -o test.html | cat
```
Results in no output the page contents going to `test.html` instead of stdout for me. Did you test this?